lib/registration.pm

```diff
@@ -223,13 +217,8 @@ sub fill_in_registration_data {
         while ($counter--) {
             die 'Registration repeated too much. Check if SCC is down.' if ($counter eq 1);
             assert_screen(\@tags);
-            if (match_has_tag("import-untrusted-gpg-key")) {
-                if (check_var("IMPORT_UNTRUSTED_KEY", 1) || check_screen(\@known_untrusted_keys, 0)) {
```

Comment:
Wouldn't it also make sense to call `handle_untrusted_gpg_key` if `IMPORT_UNTRUSTED_KET` is set to true?

Reply:
IMPORT_UNTRUSTED_KEY is not needed, just redundant to tag `import-known-untrusted-gpg-key`, will be removed. When IMPORT_UNTRUSTED_KEY is used unknown key could be blindly  imported